script/run

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -ex
 
-mix phoenix.server
+DB_URL="postgres://postgres@localhost:5432/changelog_dev" mix phoenix.server
```

Comment:
I'm pretty sure this would jack up our production deploy. @gerhard do you agree?